src/app/missions/detail/detail.component.css

```diff
@@ -23,6 +23,10 @@
     background-color: unset;
 }
 
+#mission-display, #mission-prereqs {
+    text-align: center;
```

Comment:
I'd prefer `flexbox` or `margin: auto` to center these larger blocks, rather than text layout attributes